Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml

```diff
@@ -21,7 +21,7 @@
     ClothingNeckScarfStripedRed: 3
     ClothingOuterArmorBasicSlim: 2 
     ClothingEyesBlindfold: 1
-    ClothingShoesBootsCombat: 1
+    ClothingShoesBootsCombatFilled: 1
```

Comment:
This change is unnecessary please revert.

Reply:
I'm least attached to this part of it, sure no problem